templates/archive-faq.php

```diff
@@ -1,23 +1,58 @@
 <?php get_header(); ?>
 
 <article>
-	<div class="container my-5 ucf-faq-list">
-		<h1 class="ucf-faq-title mb-4">Frequently Asked Questions</h1>
+	<div class="<?php if( get_option( 'ucf_faq_include_athena_classes' ) ) echo "container my-5 "; ?>ucf-faq-list">
+		<h1 class="ucf-faq-title<?php if( get_option( 'ucf_faq_include_athena_classes' ) ) echo " mb-4"; ?>">Frequently Asked Questions</h1>
 		<?php
 		if ( have_posts() ) {
-			while ( have_posts() ) {
-				the_post();
+
```

Comment:
I would move everything down to line 26 into a function stored within a separate file. Keeping data fetching code out of the template is helpful in keeping things organized.